atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -306,9 +306,13 @@ private boolean trySchemaMutationUnlockOnce(long perOperationNodeId) {
         try {
             result = clientPool.runWithRetry(client -> {
                 Column existingColumn = queryExistingLockColumn(client);
+                long existingLockId = getLockIdFromColumn(existingColumn);
 
-                Preconditions.checkState(isValidColumnForLockId(existingColumn, perOperationNodeId),
-                        "Trying to unlock unowned lock");
+                if (existingLockId != GLOBAL_DDL_LOCK_CLEARED_ID) {
```

Comment:
This seems odd. Why aren't we doing a return instead? Make this runWithRetry return whether it was successful or not instead.